packages/askui-nodejs/package.json

```diff
@@ -14,7 +14,7 @@
         "lint": "eslint --cache --fix --max-warnings 0 \"./**/*.{js,ts}\"",
         "lint:staged": "lint-staged",
         "test": "jest '/src'",
-        "postinstall": "$npm_package_bin_askui-postinstall"
+        "postinstall": "node -e \"require('./bin/askui-postinstall')\""
```

Comment:
just quick question, can'T this be replaced with npx askui-postinstall ?

Reply:
Nope, as `askui-postinstall` would not be found in this case

Reply:
thanks for the responce